docs/user/06-conditions.md

```diff
@@ -0,0 +1,37 @@
+# Telemetry CR conditions
+
+This section describes the possible states of the Telemetry CR. There are conditions type `LogComponentsHealthy`, `TraceComponentsHealthy` and `MetricComponentsHealthy` representing state of each of the subcomponents. The state of the Telemetry CR is derived from combined state of all the subcomponents.
+
```

Comment:
In each of the following three tables, which value does the column "Condition Type" add? It's identical in all lines. Couldn't this simply be the headline of each section, or an introductory sentence?

Reply:
You are right, it does not make much sense. Will remove it together with the numbers